packages/client/modules/teamDashboard/components/AgendaListAndInput/AgendaListAndInput.tsx

```diff
@@ -31,30 +35,70 @@ interface Props {
   dashSearch?: string
   gotoStageId?: ReturnType<typeof useGotoStageId>
   isDisabled?: boolean
+  meeting: AgendaListAndInput_meeting | null
   team: AgendaListAndInput_team
-  meetingId?: string
+}
+
+const getAgendaItems = (meeting) => {
+  if (!meeting) return null
+  const agendaItemsPhase = meeting.phases!.find(
+    (phase) => phase.phaseType === NewMeetingPhaseTypeEnum.agendaitems
+  )!
+  if (!agendaItemsPhase.stages) return null
+  return agendaItemsPhase.stages.map((stage) => stage.agendaItem)
 }
 
 const AgendaListAndInput = (props: Props) => {
-  const {dashSearch, gotoStageId, isDisabled, team, meetingId} = props
+  const {dashSearch, gotoStageId, isDisabled, team, meeting} = props
+  const endedAt = meeting?.endedAt
+  const agendaItems = endedAt ? getAgendaItems(meeting) : team.agendaItems
```

Comment:
while they should be the same, it might be more expressive to say `getAgendaItems(meeting) || team.agendaItems`.

if there's a meeting, grab the agenda from the meeting. otherwise, grab it from the team